inc/class-wpseo-image-utils.php

```diff
@@ -243,7 +243,11 @@ public static function get_file_size( $image ) {
 		// If the file size for the file is over our limit, we're going to go for a smaller version.
 		// @todo Save the filesize to the image metadata.
 		// phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged -- If file size doesn't properly return, we'll not fail.
-		return @filesize( self::get_absolute_path( $image['path'] ) );
+		if(file_exists(self::get_absolute_path( $image['path'] ))) {
```

Comment:
Please fix the code style here (I'd do it myself but can't write to your branch):

```
if ( file_exists( self::get_absolute_path( $image['path'] ) ) ) {
```